astroid/helpers.py

```diff
@@ -138,7 +141,11 @@ def object_issubclass(node, class_or_seq, context=None):
     return _object_type_is_subclass(node, class_or_seq, context=context)
 
 
-def safe_infer(node, context=None):
+# TODO: PY310: Remove the disable after InferenceResult no longer uses Union
+def safe_infer(  # pylint: disable=consider-alternative-union-syntax
```

Comment:
What has the `Union` in `InferenceResult` to do with the syntax here?

Reply:
This crashes without the `Union`. `pylint` doesn't recognise this. See:
https://github.com/PyCQA/astroid/runs/6956094603?check_suite_focus=true

Or at least, I assumed it's because of the both `Union` and `|`.

Reply:
Try adding `from __future__ import annotations`

Reply:
😓